### DISCUSSION TITLE
发送卡片消息

### DISCUSSION THREAD

**Discussion by xorange1997** [simple-robot/Discussions#129] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

作者 您好，请问一下能够支持发送这种卡片消息吗？如果能还希望有个事例，万分感谢 
![image](https://user-images.githubusercontent.com/38392442/236593407-3c345df5-3e6b-4a29-ad3d-9db66c177e82.png)

**Comment by ForteScarlet** [comment]
mirai中，卡片消息相关信息都在 [LightApp (RichMessage) ](https://github.com/mamoe/mirai/blob/dev/mirai-core-api/src/commonMain/kotlin/message/data/RichMessage.kt) 中，通常为 `JSON` 或 `XML` 格式。
但是卡片的样式、内容都无法确定，可以通过发送给bot并接收后，尝试自行分析其content中的格式。

> `LightApp` 链接参考自：[mirai开发手册](https://github.com/mamoe/mirai/blob/dev/docs/Messages.md#%E6%B6%88%E6%81%AF%E7%B1%BB%E5%9E%8B)

当构建出 mirai 的 `LightApp` 之后，参考 [Discussions#47](https://github.com/orgs/simple-robot/discussions/47) 来转化为simbot的类型并发送。
